.github/workflows/backend-prod-cd.yml

```
@@ -79,233 +79,34 @@ jobs:
-          	if [ -z "$CHECK_PID" ]; then
-          		echo "기존 서버 종료됨"
-          	else
-          		echo "기존 서버 종료되지 않아 강제 종료 수행... PID=$CHECK_PID"
-          		sudo kill -9 $CHECK_PID || true
-          		sleep 5
-          	fi
-          else
-          	echo "포트 80은 사용 중이지 않음"
-          fi
+    if: needs.detect-idle.outputs.idle-server == 'prod-app-2'
+    uses: .github/workflows/prod-app-2-cd.yml
```

actionlint (1.7.7)

reusable workflow call ".github/workflows/prod-app-2-cd.yml" at "uses" is not following the format "owner/repo/path/to/workflow.yml@ref" nor "./path/to/workflow.yml". see https://docs.github.com/en/actions/learn-github-actions/reusing-workflows for more details

(workflow-call)